src/tools/get-bootstrap-config.ts

```diff
@@ -0,0 +1,26 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+
+import { bootstrapConfigCommand } from './commands/bootstrap-config';
+import { process } from './node-shim';
+
+/**
+ * Script to run the bootstrap-config command directly.
```

Comment:
The command is built in the way that it is in order to be integrated into a higher-order CLI structure, thus we define the action as a `Command` and separate it from actually running it.